src/SFA.DAS.Payments.RequiredPayments.Tests.Specs/StepDefinitions/CalculatorStepDefinitions.cs

```diff
@@ -0,0 +1,46 @@
+namespace SFA.DAS.Payments.RequiredPayments.Tests.Specs.StepDefinitions
+{
+
+
+    [Binding]
+    public sealed class CalculatorStepDefinitions
```

Comment:
As commented on the feature file, is this needed?

Reply:
Removed